contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_present_in_other_table.py

```diff
@@ -0,0 +1,227 @@
+     the values of CUSTOMER_ID in order_table_1 are present in the CUSTOMER_ID column of customer_table.
+
+    "order_table_1": {
+         "ORDER_ID": ["aaa", "bbb", "ccc"],
+         "CUSTOMER_ID": [1, 1, 3],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+
+     However, in the second example, referential integrity is not preserved, because there are two values (4 and 5) in the CUSTOMER_ID
+     column of order_table_2 that are not present in the CUSTOMER_ID column of customer_table.
+
+     "order_table_2": {
+         "ORDER_ID": ["ddd", "eee", "fff"],
+         "CUSTOMER_ID": [1, 4, 5],
+     }
+     "customer_table": {
+         "CUSTOMER_ID": [1, 2, 3],
+
+     }
+     ExpectColumnValuesToBePresentInAnotherTable will PASS for example 1 and FAIL for example 2.
+
+     Args:
+         template_dict: dict containing the following keys:
+             foreign_key_column: foreign key-column of current table that we want to validate.
+             foreign_table: foreign table.
+             primary_key_column_in_foreign_table: key column for primary key in foreign table.
+    """
+
+    library_metadata = {
+        "maturity": "experimental",
+        "tags": ["table expectation", "multi-table expectation", "query-based"],
+        "contributors": [
+            "@great_expectations",
+        ],
+        "requirements": [
+            "sqlalchemy",
+            "snowflake-sqlalchemy",
+            "snowflake-connector-python",
+        ],
+        "has_full_test_suite": False,
+        "manually_reviewed_code": True,
+    }
+
+    metric_dependencies = ("query.template_values",)
+    template_dict: dict
+    query = """
+        SELECT count(1) FROM (
+        SELECT a.{foreign_key_column}
+        FROM {active_batch} a
+        LEFT JOIN {foreign_table} b
+            ON a.{foreign_key_column} = b.{primary_key_column_in_foreign_table}
+        WHERE b.{primary_key_column_in_foreign_table} is NULL)
+        """
+    success_keys = (
+        "template_dict",
+        "query",
+    )
+    domain_keys = ("query", "batch_id", "row_condition", "condition_parser")
+
+    default_kwarg_values = {
+        "catch_exceptions": False,
+        "meta": None,
+        "query": query,
+    }
+
+    @classmethod
+    @override
+    @renderer(renderer_type="renderer.prescriptive")
+    def _prescriptive_renderer(
+        cls,
+        configuration: Optional[ExpectationConfiguration] = None,
+        result: Optional[ExpectationValidationResult] = None,
+        runtime_configuration: Optional[dict] = None,
+    ) -> List[RenderedStringTemplateContent]:
+        runtime_configuration = runtime_configuration or {}
+        styling = runtime_configuration.get("styling")
+
+        template_dict = configuration.kwargs.get("template_dict")
+
+        template_str = "No values in the column $foreign_key_column are expected to be missing in table $foreign_table 's $primary_key_column_in_foreign_table column."
+
```

Comment:
This is the current template string. Would appreciate feedback here.

Reply:
Just throwing an option out: `All values in column $foreign_key_column are present in column $primary_key_column_in_foreign_table of  table $foreign_table.`

Reply:
Posted a suggestion on the line, let me know what you think!

Reply:
![Screenshot 2023-12-20 at 12 12 13 PM](https://github.com/great-expectations/great_expectations/assets/25670697/d91f250f-1c76-40fd-8dcc-7afa61c60d96)

I'm not sure how this will fit in with "Observed Value" as `0` when the Expectation passes and non-`0` when the Expectation fails. This is where the original "no values" comes into play, but maybe the results should be self explanatory? @anthonyburdi

Reply:
Good point. How about adding a parenthetical afterward, something like "(observed value is the number of missing items)"?

Also I like Jenny's suggested text better. Mine was trying to match other expectations that show column name first but it's clearer how @JennyTee wrote it.

Reply:
Thank you @anthonyburdi. I'll update and ask the # product channel for some feedback before merging :)